rust/src/shard_generated.rs

```diff
@@ -290,8 +292,11 @@ pub mod sedpack {
                     }
                 }
                 pub enum ShardOffset {}
-                #[derive(Copy, Clone, PartialEq)]
 
+
+                // Added Yokeable to the autogenerated code.
+                use yoke::Yokeable;
+                #[derive(Yokeable, Copy, Clone, PartialEq)]
```

Comment:
nit, you can use the fully qualified name in the derive to avoid "polluting" the scope
```suggestion
                #[derive(Copy, Clone, PartialEq, yoke::Yokeable)]
```